config/templates/cloud-config-worker

```diff
@@ -194,6 +194,29 @@ coreos:
         WantedBy=kubelet.service
 {{ end }}
 
+    - name: cfn-signal.service
+      command: start
+      content: |
+        [Unit]
+        Wants=kubelet.service docker.service
+        After=kubelet.service
```

Comment:
This way, does `cfn-signal.service` get started only after `kubelet.service` successfully started?